sct.py

```diff
@@ -1351,5 +1356,33 @@ def clean_aws_mocks(test_id: str | None, all_mocks: bool, verbose: bool, dry_run
     AwsMock.clean(test_id=test_id, all_mocks=all_mocks, verbose=verbose, dry_run=dry_run)
 
 
+@cli.command("generate-pt-report", help="Generate parallel timelines representation for the SCT test events")
+@click.option("-t", "--test-id", help="Test ID to search in sct-results")
+@click.option("-d", "--logdir", help="Directory with sct-results folder")
+def generate_parallel_timelines_report(logdir: str | None, test_id: str | None) -> None:
```

Comment:
Please, add `add_file_logger()` at the beginning of the command.

Reply:
Added